2024.1 (#32)
Mte90/Contribute-to-opensource-the-right-way: manuscript/chapter5-what-you-can-do.md

* Privacy is important because it's possible to identify you even if you are navigating in private mode on the internet due to the proprietary feature of your browser full of bugs.
* Open source doesn't mean it is free, but that the project is under a license allowing specific things. So it's sometimes possible to get the code by paying it for example, like in WordPress plugin/theme ecosystem.

> About Privacy

> > It is a topic that growth a lot in the past years and there are tons of stories about why you should care of it. I want to share this to you [How the Pentagon Learned to Use Targeted Ads to Find Its Targets—and Vladimir Putin](https://www.wired.com/story/how-pentagon-learned-targeted-ads-to-find-targets-and-vladimir-putin/), to understand how much bigger can be.

Reviewer: ```suggestion
> It is a topic that grew a lot in the past years […]
```
